src/integration-test/java/com/commercetools/project/sync/CliRunnerIT.java

```diff
@@ -303,6 +306,131 @@ void run_WithSyncAsArgumentWithAllArg_ShouldExecuteAllSyncers() {
     }
   }
 
+  @Test
+  void run_WithUpdatedCustomer_ShouldSyncCustomersAndStoreLastSyncTimestampsAsCustomObject() {
+    ZonedDateTime lastModifiedTime = null;
+    // preparation
+    try (final SphereClient targetClient = createClient(CTP_TARGET_CLIENT_CONFIG)) {
```

Comment:
Test fixture contains one customer in the source project, so it's not easy to ensure if the delta working correctly, I would expect a scenario at least one same customer both in target and source with runned delta sync.

Reply:
Hi @ahmetoz, I am first synching the customer from source project to target, And then I am updating the customer in Source and running the Sync program one more time, Later I assert to check if the target Customer also updated properly.

Reply:
I know your scenario, but if we have one resource it's hard to ensure in the future to avoid bugs with tests.. so how to make sure, if it's a full sync or delta sync... again, I would expect a state you have already a fixture preparing full sync and then running delta sync, what do you think ?

Reply:
Okay I understood your point now. I will add one more customer and update only one customer to verify delta sync. Is that okay?

Reply:
@ahmetoz , This case is not possible to implement because of the code in the below code, existing delay of 2 minutes(LastSyncTimestamp).
https://github.com/commercetools/commercetools-project-sync/blob/15a957ce9f422913165712cd638e2ee552174283/src/main/java/com/commercetools/project/sync/Syncer.java#L222
But I am asserting LastSyncTimestamp, which verifies that delta sync has run.